lib/android/app/build.gradle

```diff
@@ -66,7 +66,11 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_1_8
+        if (reactNativeMinorVersion() >= 73) {
+            jvmTarget = JavaVersion.VERSION_17
+        } else {
+            jvmTarget = JavaVersion.VERSION_11
+        }
```

Comment:
ok, cool. Did you somehow check that it really does work with RN 71?

Reply:
Yes